packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -50,9 +60,65 @@ const getContainedSize = (img: HTMLCanvasElement): Rect => {
   }
   const x = (imgClientWidth - width) / 2;
   const y = (imgClientHeight - height) / 2;
-  return { x: x, y: y, width: width, height: height };
+  return { action: '', x: x, y: y, width: width, height: height };
 };
 
+function drawRect(rect: Rect, ctx: CanvasRenderingContext2D, feedbackId: string, scale: number = 1): void {
+  const scaledX = rect.x * scale;
+  const scaledY = rect.y * scale;
+  const scaledWidth = rect.width * scale;
+  const scaledHeight = rect.height * scale;
+
+  switch (rect.action) {
+    case 'highlight': {
+      // creates a shadow around
+      ctx.shadowColor = 'rgba(0, 0, 0, 0.7)';
+      ctx.shadowBlur = 50;
+
+      // draws a rectangle first so that the shadow is visible before clearing
+      ctx.fillStyle = 'rgb(0, 0, 0)';
+      ctx.fillRect(scaledX, scaledY, scaledWidth, scaledHeight);
+      ctx.clearRect(scaledX, scaledY, scaledWidth, scaledHeight);
+
+      // Disable shadow after the action is drawn
+      ctx.shadowColor = 'transparent';
+      ctx.shadowBlur = 0;
+
+      // draws outline around rectangle in the same colour as the submit button
+      let strokeColor;
+      const sentryFeedback = DOCUMENT.getElementById(feedbackId);
+      if (sentryFeedback) {
+        const computedStyle = getComputedStyle(sentryFeedback);
+        strokeColor =
+          computedStyle.getPropertyValue('--button-primary-background') ||
+          computedStyle.getPropertyValue('--accent-background');
+      }
```

Comment:
Instead of passing in feedbackId, should probably pass in `strokeColor` so that we calculate it just once